src/tools/compiletest/src/runtest/rustdoc_json.rs

```diff
@@ -16,13 +16,12 @@ impl TestCx<'_> {
             self.fatal_proc_rec("rustdoc failed!", &proc_res);
         }
 
-        let root = self.config.find_rust_src_root().unwrap();
         let mut json_out = out_dir.join(self.testpaths.file.file_stem().unwrap());
         json_out.set_extension("json");
         let res = self.run_command_to_procres(
             Command::new(self.config.jsondocck_path.as_ref().unwrap())
                 .arg("--doc-dir")
-                .arg(root.join(&out_dir))
+                .arg(&out_dir)
```

Comment:
This currently is useless because `out_dir` is an absolute path, which `root.join(&out_dir)` will just become `out_dir`. This would be actually *wrong* if `out_dir` was a relative path, because build dir is not necessarily under source root (such as in CI).